### PR TITLE
Cleanup whitespace

### PR DESCRIPTION
**Pull request by tamird**
r? @alexcrichton

### CODE REVIEW

File: src/doc/trpl/error-handling.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 % Error Handling
 
-> The best-laid plans of mice and men
```
Comment (killercup) [inline src/doc/trpl/error-handling.md]: These spaces are correct – they are needed to force the soft line break in markdown (`<br/>`).

Please make sure the other markdown files don't contain spaces like these before removing them.

Comment (tamird) [inline src/doc/trpl/error-handling.md]: Heh, this is only instance of this usage. Fixed.

Comment (killercup) [inline src/doc/trpl/error-handling.md]: Haha, good. It was me who added that one some time ago ;)

**Comment by alexcrichton** [comment]
@bors: r+ f576579 rollup

Thanks!

**Comment by Manishearth** [comment]
Needs https://github.com/Manishearth/rust/commit/dcec88cca3f4688241872e21d5a08f339f938879 to pass tests.

**Pull request merged**